css/templates/lune/pages/_landing.css

```diff
@@ -1,3 +1,157 @@
 /* //////////////////////////////////////////////////////
    LANDING PAGE
-///////////////////////////////////////////////////////*/
\ No newline at end of file
+///////////////////////////////////////////////////////*/
+
+:root {
+  --lune-landing-title-offset-horizontal-narrow: -6%;
+  --lune-landing-title-offset-horizontal-tablet: -32%;
+  --lune-landing-title-offset-horizontal-wide: -38%;
+
+  --lune-landing-page-height: 100vh;
+  --lune-landing-page-min-height: 37.5rem; /* 600px */
+}
+
+
+/*******************************************************************************
+ * Tweaking the delay for the entering animations defined in _pages.css
+ ******************************************************************************/
+.lune-landing-title-middle {
+  animation-delay: calc(var(--animation-delay-start) +
+      var(--animation-delay-step));
+}
+
+.lune-landing-title-bottom {
+  animation-delay: calc(var(--animation-delay-start) +
+      2 * var(--animation-delay-step));
+}
+
+.lune-page-landing .lune-component-cta {
+  animation-delay: calc(var(--animation-delay-start) +
+      3 * var(--animation-delay-step));
+}
+
+/*******************************************************************************
+ * Layout & typography
+ ******************************************************************************/
+.lune-page-landing {
+  padding-top: 0;
+  height: auto;
+}
+
+.lune-landing-section {
+  height: var(--lune-landing-page-height);
+  min-height: var(--lune-landing-page-min-height);
+}
+
+.lune-landing-wrapper {
+  top: var(--navbar-height);
+}
+
+.lune-page-landing .lune-component-cta {
+  min-width: 16rem;
+}
+
+/*
+ * When there is enough estate in the viewport, set a fixed height for the page
+ * and fixed position on the lower section, in order to achieve a "reveal" effect.
+ * Threshold is 600px, same as --lune-landing-page-height.
+ */
+@media (min-height: 37.5rem) {
+  .lune-page-landing {
+    height: calc(2 * var(--lune-landing-page-height));
+  }
+
+  .lune-landing-section.lune-theme-dark {
+    position: fixed;
+  }
+
+  /*
+   * Only apply this enhancement if clip-path is supported
+   * and the viewport is high enough.
+   */
+  @supports (clip-path: inset(1px 0)) {
```

Comment:
Progressive enhancement applied only when `clip-path` is supported.

Note that in Chrome the result is a bit buggy — a ticket has been raised here https://bugs.chromium.org/p/chromium/issues/detail?id=740962